### DISCUSSION TITLE
Game crashes when i open the settings menu

### DISCUSSION THREAD

**Discussion by Quills245** [ppy/osu#27578] [Q&A] [answered]
After the last update (2024.312.0) The game started crashing right after i open the options menu, here a video of how it does:

https://github.com/ppy/osu/assets/99212613/331e4b01-79ab-41d4-b297-b12d36103362

Logs:
[logs.zip](https://github.com/ppy/osu/files/14565321/logs.zip)

OS: arcolinux (arch based) kernel: 6.7.8-arch1-1 x86_64

**Comment by LumpBloom7** [comment, marked as answer]
the culprit seems to be Hitokori ruleset, please remove the ruleset for now. you can reinstall it once an update is available.

**Reply by Quills245** [reply]
oh, the rulesets right, that makes sense, forgot that they add settings, let me test that

**Reply by Quills245** [reply]
It worked! should have though of that haha, thanks there, sorry for the inconvenience

**Reply by ItsAyaya** [reply]
I had this problem too, thanks so much!
